src/talk/renderer/TitleBar/components/MainMenu.vue

```diff
@@ -73,11 +108,19 @@ const openInWeb = () => window.open(generateUrl(getCurrentTalkRoutePath()), '_bl
 			</template>
 			{{ t('talk_desktop', 'App settings') }}
 		</NcActionButton>
-		<NcActionButton @click="showHelp">
+		<NcActionButton close-after-click @click="showHelp">
 			<template #icon>
 				<IconInformationOutline :size="20" />
 			</template>
 			{{ t('talk_desktop', 'About') }}
 		</NcActionButton>
 	</NcActions>
 </template>
+
+<style lang="scss" scoped>
+.menu-notification-badge {
+	position: absolute;
+	top: 1px;
+	right: 1px;
+}
```

Comment:
- `.icon-group` to position dot relative to the icon and not the button
- Logical properties to support RTL languages
- `background`, `box-shadow` and `border-radius` to have a visual separation from the menu icon (like a blue outline)

```suggestion
.icon-group {
	position: relative;
}

.menu-notification-badge {
	position: absolute;
	inset-block-start: 0;
	inset-inline-end: -10%;
	background-color: var(--color-background-plain);
	border-radius: 50%;
	box-shadow: 0 0 0 1px var(--color-background-plain);
}
```